source/_posts/2018-07-21-release-74.markdown

```diff
@@ -30,7 +30,7 @@ To try it out today, [read these instructions](https://developers.home-assistant
 
 We keep seeing [great examples](https://twitter.com/home_assistant/status/1019579208622845953) of UIs built with Lovelace. Follow us on social media ([FB](https://www.facebook.com/homeassistantio/?ref=bookmarks), [Twitter](https://twitter.com/home_assistant)) where we will keep sharing great examples.
 
-For the Lovelace changes in this release, check out the [changelog](/lovelace/changelog/). To help our development and design teams, we've also introduced a [Lovelace card gallery](https://home-assistant-lovelace-gallery.netlify.com/).
+For the Lovelace changes in this release, check out the [changelog](/lovelace/changelog/). To help our development and design teams, we've also introduced a [Lovelace card gallery](https://www.awesome-ha.com/).
```

Comment:
This is an incorrect change. The Awesome list is NOT the Lovelace card gallery.

Reply:
/CC @fabaff

Reply:
@frenck apologies. How should I back out the changes? Create another PR with the revision back to the original?

Reply:
If you'd could create another PR that removed that part (the Gallery is not in use anymore), that would be awesome @claughinghouse 👍

Reply:
No, it's not but it's better to send people to a place that exists than to a 404.

https://home-assistant-lovelace-gallery.netlify.com/ is no longer reachable.

Reply:
Agreed. My initial bet would be removing that link & text all together.